usaspending_api/references/v2/views/city.py

```diff
@@ -69,23 +69,18 @@ def prepare_search_terms(request_data):
 
 
 def create_elasticsearch_query(return_fields, scope, search_text, country, state, limit):
-    query_string = create_es_search(scope, search_text, country, state)
+    query = create_es_search(scope, search_text, country, state)
     # Buffering the "group by city" to create a handful more buckets, more than the number of shards we expect to have,
     # so that we don't get inconsistent results when the limit gets down to a very low number (e.g. lower than the
     # number of shards we have) such that it may provide inconsistent results in repeated queries
     city_buckets = limit + 100
     if country != "USA":
-        aggs = {"states": {"terms": {"field": return_fields[2], "size": 100}}}
+        aggs = {"field": return_fields[2], "size": 100}
     else:
-        aggs = {"states": {"terms": {"field": return_fields[1], "size": 100}}}
-    query = {
-        "_source": return_fields,
-        "size": 0,
-        "query": {"bool": {"filter": {"bool": query_string}}},
-        "aggs": {
-            "cities": {"terms": {"field": "{}.keyword".format(return_fields[0]), "size": city_buckets}, "aggs": aggs}
-        },
-    }
+        aggs = {"field": return_fields[1], "size": 100}
+    agg_values = {"field": "{}.keyword".format(return_fields[0]), "size": city_buckets}
+    agg_key = A("terms", **agg_values)
+    query.aggs.bucket("cities", agg_key).pipeline("states", A("terms", **aggs))
```

Comment:
Can you change the second terms aggregation to be a `.bucket()` instead of `.pipeline()` since that is the type of aggregation that it is. They have the same effect in this case since the only difference in the DSL is what they return, but I can see this possibly causing confusion.

Reply:
sure thing